readthedocs/rtd_tests/tests/test_privacy_urls.py

```diff
@@ -382,6 +382,7 @@ def setUp(self):
         }
         self.response_data = {
             'build-concurrent': {'status_code': 403},
+            'build-reset': {'status_code': 403},
```

Comment:
Not sure if this is useful anymore. Might be owrth removing this automation if we feel we have good test coverage over this stuff.

Reply:
Not sure if they are that useful, but it doesn't hurt having more tests p: